test/e2e_node/services/kubelet.go

```diff
@@ -89,11 +89,11 @@ func RunKubelet() {
 
 const (
 	// Ports of different e2e services.
-	kubeletReadOnlyPort = "10255"
+	kubeletHealthzPort = "10248"
 	// KubeletRootDirectory specifies the directory where the kubelet runtime information is stored.
 	KubeletRootDirectory = "/var/lib/kubelet"
 	// Health check url of kubelet
-	kubeletHealthCheckURL = "http://127.0.0.1:" + kubeletReadOnlyPort + "/healthz"
+	kubeletHealthCheckURL = "http://127.0.0.1:" + kubeletHealthzPort + "/healthz"
```

Comment:
Can you dedupe this with the one in util.go? I'm not sure which direction the dependencies go, but I'd just pick one to export.